app/static/pages/volunteer/VolunteerInfoDefinitions.js

```diff
@@ -97,19 +92,24 @@ oppiaFoundationWebsite.constant('VOLUNTEER_INFO', {
     position: 'Researcher and Lesson Creator'
   }],
 
-  marketing: [{
+  lesson_creation: [{
+    fullName: 'Anmol Shukla',
+    profilePictureImageFilename: 'anmol.jpg',
+    professionalQuote: 'I strongly identify with Oppia\'s objective of ' +
+    'providing free, enjoyable and accessible education to all.',
+    position: 'Developer, Machine Learning'
```

Comment:
Change to: "Developer and Lesson Creator"

Reply:
Done!